pkg/virt-controller/watch/vmi/vmi_test.go

```diff
@@ -1243,6 +1247,7 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 
 			controller.Execute()
 			expectVMIScheduledState(vmi)
+			Expect(netStatusUpdated).To(BeTrue())
```

Comment:
You probably need to add text that explains what failed, otherwise it will be very difficult to understand it.